Readme.md

```diff
@@ -155,6 +155,12 @@ These folders are installed read-only and should not be changed:
 * `Resources`: icons that are theme-agnostic
 * `Wii`: default Wii NAND contents
 
+## Packaging and udev
+In the Data folder there is a udev rule file for official GameCube controller
```

Comment:
Maybe start with "The Data folder contains a udev rule file..." instead? For a non-native speaker plagued by german colleagues literally translating things, this sounds too germanglished to me...